scripts/cla-query

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python
+"""Lauchpad IDs to query to see if userid has signed CLA."""
+
+from launchpadlib.launchpad import Launchpad
+from os.path import basename
+import sys
+from argparse import ArgumentParser
+
+LP_INSTANCE = "production"
+CLA_GROUPS = ["~canonical-server"]
+
+
+def get_parser():
+    parser = ArgumentParser(description=__doc__)
+    parser.add_argument(
+        "lpid", type=str,
+        help=__doc__)
+    return parser
+
+
+def main():
+
+    parser = get_parser()
+    args = parser.parse_args()
+    print('Logging into Launchpad')
+    lp = Launchpad.login_anonymously(
+        "cla-validator", LP_INSTANCE)
+    cla_signed = set()
+    cla_notsigned = []
+    member_names = set()
+    projects = ['~contributor-agreement-canonical', '~canonical-server']
```

Comment:
I think I/you probably intended to use CLA_GROUPS here.

Reply:
Updated